include/deal.II/non_matching/mapping_info.h

```diff
@@ -552,7 +625,7 @@ namespace NonMatching
     /**
      * Offset to point to the first unit point of a cell/face.
      */
-    AlignedVector<unsigned int> unit_points_index;
+    std::vector<unsigned int> unit_points_index;
```

Comment:
This is correct. But was there are particular (performance) reason for this change?

Reply:
Actually, mainly because of pretty printing in gdb. For indices `AligendVector` does not make sense but we also use it in MatrixFree MappingInfo that's why I used it here in the first place.